.github/workflows/firebase-hosting-merge.yml

```diff
@@ -14,6 +14,8 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
We should add `gob` as a trigger branches, right?

Reply:
Add it to checkSite.yml

Reply:
Done. checkSite is triggered on every branch.